### ISSUE TITLE
Sammanställa ringlista med utbildningsledare

### ISSUE THREAD

**Issue by Bjorn77Karlsson** [chas-academy/u02-redesign-fulhackarna#18]
Sammanställer ringlista utbildningsledare med följande information: 		
- Skolnamn
- Namn på rektor/utbildningssamordnare
- Telefonummer
- Länk till skolan

**Comment by OskarFindahl** [comment]
Klar: https://docs.google.com/spreadsheets/d/1e-9s6cjA84Tx8SQNEobZ34BgaXSrVJH2oNywjynNalY/edit#gid=0
